index.d.ts

```diff
@@ -14,29 +14,38 @@ declare module 'react-native-document-picker' {
       allFiles: '*/*';
       audio: 'audio/*';
       csv: 'text/csv';
+      doc: 'application/msword';
+      excel: 'application/vnd.ms-excel';
```

Comment:
there is `xls` and there is `xlsx` - I don't think we can cover this just by "excel" 

see https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types

this applies to all the items you have added

Reply:
Fair point, i will make some changes